Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM debian:buster-slim
```

Comment:
Any reason why we don't use official python image?

Reply:
Each Python image seemed specific to a given Python version, but what we want to do here is test all supported Python versions.  But maybe we can try a different approach.

Reply:
This was originally meant to be a drop-in replacement for the existing `Vagrantfile` solution

Reply:
Can we do the same on python's based image? e.g. choose the lowest python version we supported (`python:2.7-slim-buster`). then install others via `pyenv`.

Reply:
I think if we do that, we might as well change the whole approach where we use a different Python image per version.  Something for the future!